frontends/qt/resources/Makefile

```diff
@@ -0,0 +1,9 @@
+SIZES=16x16 32x32 48x48 64x64 128x128
+
+TARGETS=$(addsuffix /apps/bitbox.png, $(addprefix linux/usr/share/icons/hicolor/, ${SIZES}))
+
+all: $(TARGETS)
```

Comment:
Then it feels like this `all` target should also include windows and macos? Besides, only linux uses a makefile. The others have a convert.sh script. Just saying, doesn't feel consistent. I'd recommend converting this to a shell script like the others or adding the other OSes to this makefile.

Reply:
I am here to fix the Makefile in package issue, not to fix the Makefile itself :D

Reply:
While there, might as well improve it, right? :)

Reply:
I am saying that mostly cause I have no clue how proper Makefiles look like and work, so I avoid working on them. Happy if you want to improve them though.